### DISCUSSION TITLE
Meshcmd on a Synology disk station

### DISCUSSION THREAD

**Discussion by fishter** [Ylianst/MeshCentral#6212] [Q&A]
Hello,

I hope someone else has at least tried this setup so I can have some confidence that it *could* work.

I have a Synology diskstation which is running Linux x86_64 (x86_64 GNU/Linux synology_geminilake_920+).  On this diskstation I have a meshcmd and meshaction.txt which contain the details to connect to another server and create a route for an ssh connection. I've tested this Username/Password combination on other platforms (Linux and Windows) with the same meshaction.txt and it works perfectly.

Is there another error hiding behind the Invalid Usename/Password that I can't see?  I've tried with a -v flag to get more verbose output, but there is no additional output.

I've tried some other compilations of meshcmd, but none worked any better than the default Linux x86_64 version.

**Comment by si458** [comment]
Do u get any output at all when running the meshcmd?
Does it hang?
Close unexpectedly?
Any errors?

**Reply by fishter** [reply]
There is a pause, presumably as it makes the remote connection, then it returns "Invalid Username/Password"*.
No other output.

*I'm going from memory here, but it's a single line.
